lib/src/components/NxLoadingSpinner/types.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { ReactNode } from 'react';
+import * as PropTypes from 'prop-types';
+
+export type Props = {
+  children?: ReactNode;
+  'aria-label'?: string;
```

Comment:
Are we leaving this undocumented?

Reply:
> Are we leaving this undocumented?

I added a documentation here: 5be4975dc99661e961f60b97031c63d06c1fedcd